src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -3386,22 +3638,25 @@ def test_X_innerproduct_Y(tmpdir, dist_mem):
 
 
 def test_X_innerproduct_X(tmpdir, dist_mem):
-    ''' Test that 1) the str method of LFRicXInnerproductXKern returns the
-    expected string and 2) we generate correct code for the built-in
-    operation which calculates inner product of a real-valued field X by
-    itself as innprod = innprod + X(:)*X(:).
+    '''Test that 1) the '__str__' method of 'LFRicXInnerproductXKern'
+    returns the expected string and 2) we generate correct code for
+    the built-in operation which calculates inner product of a
+    real-valued field 'X' by itself as 'innprod = innprod +
+    X(:)*X(:). 3)'. Also test the 'metadata()' method.
```

Comment:
Quotation mark should be after `X(:)` and not after `3)`. Please also remove full stop after `3)`.

Reply:
Oops again.